kfs-purap/src/main/java/org/kuali/kfs/module/purap/document/PurchasingDocumentBase.java

```diff
@@ -72,12 +69,19 @@
 import org.kuali.rice.location.api.country.Country;
 import org.kuali.rice.location.api.country.CountryService;
 
+import java.sql.Date;
```

Comment:
I think it separates the Java API imports from the non-API, and then applies alphabetical sort within the those sub-categories. Didn't recognize it till now, so maybe I'll have half a chance at finding it in the settings to disable it.